packages/flutter_tools/templates/app/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -79,6 +79,11 @@ if(PLUGIN_BUNDLED_LIBRARIES)
     COMPONENT Runtime)
 endif()
 
+install(FILES app.desktop DESTINATION "${INSTALL_BUNDLE_DATA_DIR}/applications" RENAME "{{projectName}}.desktop"
```

Comment:
Shouldn't this go to ~/Desktop, so that the app launches with an icon when run during development?

Reply:
No, the user level `.desktop` files are stored at `~/.local/share/applications`. I think `${INSTALL_BUNDLE_DATA_DIR}` points to `~/.local/share` probably. Whereas the root level `.desktop` files are stored at `/usr/share/applications`.

Reply:
> ${INSTALL_BUNDLE_DATA_DIR} points to ~/.local/share probably.

It doesn't; see the rest of the install section of this CMake file if you are interested in the details.